src/locales/cy/fields.yml

```diff
@@ -253,6 +253,15 @@ Q00069:
 Q00083:
   legend: "Beth yw 2 lythyren gyntaf enw masnachu eich cwmni?"
   hint: "Meddyliwch am enw masnachu'r cwmni rydych chi yn, neu wedi bod, yn gyfarwyddwr."
+Q00088:
+  legend: Faint oedd eich bil dŵr diweddaraf?
+  hint: null
```

Comment:
Asked Paul about this because I'm very surprised at this inconsistency

Reply:
I have updated based on the discussion of the thread on the ticket, so the english version has no hint description